examples/python-cicd-with-cloudbuilder/README.md

```diff
@@ -0,0 +1,135 @@
+# Basic Python CI With Cloud Source Repositories and Cloud Build
+
+## Overview
+This repo contains code and instructions for using Google Cloud Source Repositories and Cloud Build to execute unit tests upon code check-in.
+
+The basic steps we need to follow are:
+1. Create a new project in Cloud Source Repository and clone it to your machine
+2. Create your code and unit tests
+3. Create a Python Cloud Builder (more below)
+4. Create a cloud build trigger (more below)
+5. Create a cloudbuild.yaml file that executes your tests using your python cloud builder.
+
+## 1. Create a new project in Cloud Source Repository
+* Go to https://source.cloud.google.com/
+* Click 'Add repository'
+* Choose 'Create new repository'
+* Specify a name, and your project name
+* Follow the instructions to 'git clone' the empty repo to your workstation and use it like you would a git repo
+
+## 2. Create your code and unit tests
+Creating unit tests is beyond the scope of this README, but if you review the tests in tests/ you'll quickly get the idea. 
+I'm using pytest as the testing suite for this project. Before proceeding make sure you can run your tests from the 
```

Comment:
Try to avoid "I" in our technical docs. "Pytest is the testing suite for this project." is fine. You should also have a period after "command line". colons would be fine too if you're introducing a code block.